benches/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def reset_object_registry():
```

Comment:
I know I put something like this in other places in the test suite, might be worth culling those.

Reply:
We'll want to consolidate these in general, this is duplicated from our `tests/conftest.py` — I figure we'll tackle redundancies here in the future.